docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -393,24 +389,24 @@ The `label` property will be used for the default locale and for languages witho
 starlight({
 	sidebar: [
 		{
-			label: 'Guides',
+			label: 'Constellations',
 			translations: {
-				'pt-BR': 'Guias',
+				'pt-BR': 'Constelação',
```

Comment:
"Constelação" is the singular form, I suppose you wanted the plural form, which is "Constelações"
```suggestion
				'pt-BR': 'Constelações',
```

Reply:
You are 100% correct, thanks :raised_hands: